tests/test_polylabel.py

```diff
@@ -68,7 +37,7 @@ def test_rectangle_special_case(self):
         polygon = Polygon([(32.71997,-117.19310), (32.71997,-117.21065),
                            (32.72408,-117.21065), (32.72408,-117.19310)])
         label = polylabel(polygon)
-        self.assertEqual(label.coords[:], [(32.722025, -117.201875)])
+        self.assertEqual(label.coords[:], [(32.722025, -117.208595)])
```

Comment:
This test was added after this report (and fix): https://github.com/mapbox/polylabel/issues/3 
So it seems that the GEOS implementation has the same issue with rectangular polygons (will bring it up with GEOS to see if they are interested in the fix, though it is not strictly a bug I think)